404.html

```diff
@@ -2,12 +2,12 @@
 <html lang=en>
 	<head>
 		<title>404 - Page Not Found</title>
-		<base href="https://theforumhelpers.github.io"> <!--If making local changes, change the base tag to point to how your filesystem is set up-->
 		<link rel="shortcut icon" type="image/png" href="resources/favicon.png">
 		<link rel="stylesheet" type="text/css" href="resources/stylesheet.css">
 	</head>
 	<body>
 		<header id="header"></header>
+		<base href="https://theforumhelpers.github.io"> <!--If making local changes, change the base tag to point to how your filesystem is set up-->
```

Comment:
Wasn't the whole point of this to remove the base element?

Reply:
See the comment by @gosoccerboy5

Reply:
@Accio1 Oops. Was I wrong about that? I thought without the \<base> element, #141 would pop up again, or if we used /resources/stylesheet.css then it would direct to the base of their filesystem.. am I wrong somewhere?